frontend/src/helpers/renderHelpers.tsx

```diff
@@ -0,0 +1,19 @@
+import { SectionBox } from '../components/common';
+import { useTypedSelector } from '../redux/reducers/reducers';
+
+export default function DetailsViewRenderMapper(props: { resource: any }) {
```

Comment:
I'd prefer a name that's more direct and thus easier to understand, so we can read:
`<DetailsViewPluginsRenderer />`